net-test/bin/start.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+
+MODE="$1"
+set -uo pipefail
+
+BITCOIN_LOGFILE="/mnt/bitcoin.log"
+BITCOIN_NEON_CONTROLLER_LOGFILE="/mnt/bitcoin-neon-controller.log"
+STACKS_MASTER_LOGFILE="/mnt/stacks-node-master.log"
+STACKS_MINER_LOGFILE="/mnt/stacks-node-miner.log"
+STACKS_FOLLOWER_LOGFILE="/mnt/stacks-node-follower.log"
+FAUCET_LOGFILE="/mnt/faucet.log"
+
+BITCOIN_CONF="/etc/bitcoin.conf"
+BITCOIN_CONTROLLER_CONF="/etc/bitcoin-neon-controller.toml"
+STACKS_MASTER_CONF="/etc/stacks-master.toml"
+STACKS_MINER_CONF="/etc/stacks-miner.toml"
+STACKS_FOLLOWER_CONF="/etc/stacks-follower.toml"
+
+STACKS_CHAINSTATE_DIR="/mnt/stacks-chainstate"
+BITCOIN_DATA_DIR="/mnt/bitcoin"
+
+STACKS_MASTER_PUBLIC_IP="127.0.0.1"
+
+FAUCET_PORT=8080
```

Comment:
I've had something running on port 8080 for years, so this would be helpful if this was configurable

Reply:
Sure, I can put that into `config.sh`

Reply:
Fixed